From 78706f3f524df5910d4960e94f2000f7c37b0462 Mon Sep 17 00:00:00 2001
From: Mateusz Gozdek <mgozdekof@gmail.com>
Date: Mon, 6 Jul 2020 11:30:50 +0200
Subject: [PATCH] Makefile: allow to modify how to link to libusbmuxd

On distros like Arch Linux, libusbmuxd is installed as libusbmuxd-2.0,
so building is currently broken. With this commit, building works again,
if one specifies USBMUXD=-lusbmuxd-2.0 parameter.

Signed-off-by: Mateusz Gozdek <mgozdekof@gmail.com>
---
 linux/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git linux/Makefile linux/Makefile
index 97a6e1f..5ff94e7 100644
--- linux/Makefile
+++ linux/Makefile
@@ -17,6 +17,7 @@ LIBAV = `pkg-config --libs --cflags libswscale libavutil`
 LIBS  =  -lspeex -lasound -lpthread -lm
 JPEG  = -I$(JPEG_INCLUDE) $(JPEG_LIB)/libturbojpeg.a
 SRC      = src/connection.c src/settings.c src/decoder*.c src/av.c src/usb.c
+USBMUXD = -lusbmuxd
 
 all: droidcam-cli droidcam
 
@@ -27,7 +28,7 @@ package: clean all
 	zip -x *.png src/ src/* Makefile -r droidcam_`date +%s`.zip ./*
 
 else
-LIBS += -lusbmuxd
+LIBS += $(USBMUXD)
 endif
 
 gresource: .gresource.xml icon2.png
-- 
2.27.0

